sessionctx/variable/sysvar.go

```diff
@@ -513,15 +512,14 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "character_set_database", mysql.DefaultCharset},
 	{ScopeNone, "have_symlink", "YES"},
 	{ScopeGlobal | ScopeSession, "storage_engine", "InnoDB"},
-	{ScopeGlobal | ScopeSession, "sql_log_off", "OFF"},
-	{ScopeNone, "explicit_defaults_for_timestamp", "ON"},
+	{ScopeGlobal | ScopeSession, "sql_log_off", "0"},
+	{ScopeNone, "explicit_defaults_for_timestamp", "0"},
```

Comment:
"1" ?

Reply:
@XuHuaiyu The default value of `explicit_defaults_for_timestamp` seems be `OFF`: https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_explicit_defaults_for_timestamp

Reply:
I've changed it to "ON" by default, because TiDB only supports the same behavior as MySQL when `explicit_defaults_for_timestamp` is ON.

Reply:
ok

Reply:
I've added a comment on this.